evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -904,24 +915,43 @@ def test_graph_partial_dependence(test_pipeline):
     go = pytest.importorskip('plotly.graph_objects', reason='Skipping plotting test because plotly not installed')
     clf = test_pipeline
     clf.fit(X, y)
-    fig = graph_partial_dependence(clf, X, feature='mean radius', grid_resolution=20)
+    fig = graph_partial_dependence(clf, X, features='mean radius', grid_resolution=20)
     assert isinstance(fig, go.Figure)
     fig_dict = fig.to_dict()
     assert fig_dict['layout']['title']['text'] == "Partial Dependence of 'mean radius'"
     assert len(fig_dict['data']) == 1
     assert fig_dict['data'][0]['name'] == "Partial Dependence"
 
-    part_dep_data = partial_dependence(clf, X, feature='mean radius', grid_resolution=20)
+    part_dep_data = partial_dependence(clf, X, features='mean radius', grid_resolution=20)
     assert np.array_equal(fig_dict['data'][0]['x'], part_dep_data['feature_values'])
     assert np.array_equal(fig_dict['data'][0]['y'], part_dep_data['partial_dependence'].values)
 
 
+def test_graph_two_way_partial_dependence(test_pipeline):
```

Comment:
Amazing!! Yes, RE Freddy's comment, I'd love to see what happens if we pass in a mixed tuple with an int and a string col. Does it work? Let's add a test for it too 😁